packages/xchain-cosmos/src/cosmos/types.ts

```diff
@@ -26,7 +27,7 @@ export type TransferParams = {
   amount: BigSource
   asset: string
   memo?: string
-  fee?: string
+  gasLimit?: BaseAmount
```

Comment:
```
gasLimit?: BaseAmount` -> `gasLimit?: BigNumber
```
(Sorry, I was wrong with a previous review comment it needs to be a `BaseAmount`, but still a `BigNumber` and not a `string`)

Reply:
ok i ended up going with this, which is what other clients use for amount/gasLimit 
```typescript
export type TransferParams = {
  privkey: proto.cosmos.crypto.secp256k1.PrivKey
  from: string
  to: string
  amount: BaseAmount
  asset: string
  memo?: string
  gasLimit?: BigNumber
}
```